app/agents/voice/automatic/__init__.py

```
@@ -339,7 +340,18 @@ async def run_normal_mode(args):
     # Audio filter configuration
-    if config.ENABLE_AIC_FILTER and config.AICOUSTICS_LICENSE_KEY:
+    if (
+        config.ENABLE_KRISP_FILTER
+        and config.KRISP_MODEL_PATH
+        and os.path.isfile(config.KRISP_MODEL_PATH)
+    ):
+        try:
+            daily_params.audio_in_filter = NoiseFilterFromKrisp(
+                model_path=config.KRISP_MODEL_PATH
+            )
+        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)